src/authentication/msal/MSALAuthenticationProviderOptions.ts

```diff
@@ -10,27 +10,3 @@
  */
 
 import { AuthenticationProviderOptions } from "../../IAuthenticationProviderOptions";
-
```

Comment:
why are we keeping the file at all?

Reply:
Kept this file with MSALBrowserAuthProviderOptions.  This file will contain options related to MSAL library or libraries (TBD).